data/Application.css

```diff
@@ -50,3 +50,13 @@ vte-terminal {
     background-color: #fff;
     color: #000;
 }
+
+.color-custom radio {
+    background-color: #fff;
+    color: #000;
```

Comment:
Yeah, I used it since there does not seem to be a dark style for the "list-add" icon. But since we are switching to "list-add-symbolic" anyways this is not an issue anymore.